internal/cmd/egctl/translate.go

```diff
@@ -55,6 +58,8 @@ func NewTranslateCommand() *cobra.Command {
 	}
 	translateCommand.PersistentFlags().StringVarP(&inType, "from", "", gatewayAPIType, getValidInputTypesStr())
 	translateCommand.PersistentFlags().StringVarP(&outType, "to", "", xdsType, getValidOutputTypesStr())
+	translateCommand.PersistentFlags().StringVarP(&output, "output", "o", yamlOutput, "One of 'yaml' or 'json'")
+	translateCommand.PersistentFlags().StringVarP(&resourceType, "type", "t", string(AllEnvoyConfigType), getValidResourceTypesStr())
```

Comment:
`--type` here is tied to `--to`, currently it only represent `xds` type, it might need to be removed in the future or reframed to `output types` and then list out all valid types per output